api/hubd/service.go

```diff
@@ -315,7 +306,21 @@ func (s *Service) CreateKey(ctx context.Context, req *pb.CreateKeyRequest) (*pb.
 		return nil, status.Errorf(codes.Internal, "mapping key type: %v", key.Type)
 	}
 
-	go s.Analytics.NewEvent(account.User.Key.String(), account.User.Email, "create_key", true, map[string]interface{}{})
+	var event analytics.Event
+	if keyType == mdb.AccountKey {
+		event = analytics.KeyAccountCreated
+	} else {
+		event = analytics.KeyUserGroupCreated
```

Comment:
Above, the only possible values for `keyType` are `mdb.AccountKey` and `mdb.UserKey`. I'm not clear on the naming of things off the top of my head, so just double checking that `mdb.UserKey` should map to `analytics.KeyUserGroupCreated` since "user" and "user group" sound like different things to me.

Reply:
yeah, nice. moved to just KeyUserCreated, no "Group"